### PR TITLE
Feat/nad 7

### PR DESCRIPTION
**Pull request by DenisAntonov33**

### CODE REVIEW

File: src/components/PlaySVG.svelte [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<script></script>
```
Comment (klikkn) [inline src/components/PlaySVG.svelte]: rename please - PlayIcon.svelte
and move all Icons to icons folder

File: public/global.css [hunk 2]
```diff
@@ -1,62 +1,147 @@
...
+  cursor: pointer;
+  font-family: Poppins;
+  font-size: 16px;
+  height: 63px;
+  line-height: 63px;
+  padding: 0 20px;
+  text-align: left;
+  width: 100%;
+}
+.user {
```
Comment (klikkn) [inline public/global.css]: why ```.user``` styles here?
can we move it to user component?
and all styles related to user below

File: public/imgs/animal-paw-print.svg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="129" height="119" viewBox="0 0 129 119">
```
Comment (klikkn) [inline public/imgs/animal-paw-print.svg]: I thought we don't need svg assets, you can save it as PawIcon.svelte

File: rollup.config.js [hunk 4]
```diff
@@ -1,47 +1,47 @@
-import svelte from 'rollup-plugin-svelte';
-import resolve from 'rollup-plugin-node-resolve';
-import commonjs from 'rollup-plugin-commonjs';
-import livereload from 'rollup-plugin-livereload';
-import { terser } from 'rollup-plugin-terser';
+import svelte from "rollup-plugin-svelte";
```
Comment (klikkn) [inline rollup.config.js:1]: Great prettier work! :D Why did you need to change this file?

File: public/global.css [hunk 5]
```diff
@@ -1,62 +1,147 @@
...
+}
+.container {
+  margin: 0 auto;
+  padding: 0 16px;
+}
+.play-btn:hover svg path {
+  fill: var(--NAD_WHITE);
+  stroke: var(--NAD_BLACK);
+}
+.profile-btn {
```
Comment (klikkn) [inline public/global.css]: why it's globbal style? it should be just in profile btn component, or we re-use it smw?

File: src/components/User.svelte [hunk 6]
```diff
@@ -0,0 +1,16 @@
+<script>
```
Comment (klikkn) [inline src/components/User.svelte]: mm, probably ```UserCard.svelte```, User is so common name for component

File: public/global.css [hunk 7]
```diff
@@ -1,62 +1,147 @@
...
+}
+
+input::placeholder {
+  /* font-family: Poppins; */
+}
+.container {
+  margin: 0 auto;
+  padding: 0 16px;
+}
+.play-btn:hover svg path {
```
Comment (klikkn) [inline public/global.css:95]: move it  from ```global.css``` to the component

**Review by klikkn — CHANGES_REQUESTED** [review]

File: public/global.css [hunk 8]
```diff
@@ -1,62 +1,147 @@
...
+  height: 63px;
+  line-height: 63px;
+  padding: 0 20px;
+  text-align: left;
+  width: 100%;
+}
+.user {
+  display: flex;
+}
+.user .avatar {
```
Comment (klikkn) [inline public/global.css]: we don't need style cascades

File: public/global.css [hunk 9]
```diff
@@ -1,62 +1,147 @@
...
+  background-color: var(--NAD_BG);
+  border-radius: 50%;
+  height: 50px;
+  margin-right: 20px;
+  width: 50px;
+}
+.user .user-info * {
```
Comment (klikkn) [inline public/global.css]: just ```.user__info``` will be better

**Review by klikkn — CHANGES_REQUESTED** [review]

**Review by klikkn — CHANGES_REQUESTED** [review]
Fix issues please

File: public/global.css (hunk 7 above)
Comment (DenisAntonov33) [reply public/global.css:95]: we cant move it from global styles, because styles in component is scoped

File: public/global.css (hunk 9 above)
Comment (DenisAntonov33) [reply public/global.css]: cascades or just css class name? .user__info looks like we're using BEM)

**Review by klikkn — APPROVED** [review]
